CHANGELOG.md

```diff
@@ -18,9 +20,27 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 ### Removed
 
+Disable pl optimizer by default to temporarily fix AMP issues ([#5163](https://github.com/PyTorchLightning/pytorch-lightning/pull/5163)
 
 ### Fixed
 
+Metric reduction with Logging ([#5150](https://github.com/PyTorchLightning/pytorch-lightning/pull/5150)
+
+
+Remove nan loss in manual optimization ([#5121](https://github.com/PyTorchLightning/pytorch-lightning/pull/5121)
+
+
+Un-balanced logging properly supported ([#5119](https://github.com/PyTorchLightning/pytorch-lightning/pull/5119)
+
+
+Fix hanging in DDP HPC accelerators ([#5157](https://github.com/PyTorchLightning/pytorch-lightning/pull/5157)
+
+
+Fix saved filename in ModelCheckpoint if it already exists ([#4861](https://github.com/PyTorchLightning/pytorch-lightning/pull/4861)
+
+
+Fix reset TensorRunningAccum ([#5106](https://github.com/PyTorchLightning/pytorch-lightning/pull/5106)
+
 
 ## [1.1.1] - 2020-12-15
```

Comment:
remove "=======" bellowL58

Reply:
Could you repoint to where this is? I see it lower down in the release notes

Reply:
it was line 58 in the next section